src/aws/osml/model_runner/tile_worker/tile_worker_utils.py

```diff
@@ -116,8 +110,10 @@ def process_tiles(
             # Use the request and metadata from the raster dataset to create a set of keyword
             # arguments for the gdal.Translate() function. This will configure that function to
             # create image tiles using the format, compression, etc. needed by the CV container.
-            gdal_translate_kwargs = create_gdal_translate_kwargs(
-                region_request.tile_format, region_request.tile_compression, raster_dataset
+            gdal_tile_factory = GDALTileFactory(
```

Comment:
You should pass in the sensor_model to the GDALTileFactory as well. That will ensure some of the geospatial metadata is updated correctly. You'll need to add it to the signature of process_tiles but it looks like the calling method has access to the sensor_model already so it shouldn't ripple too far.